src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -226,7 +226,7 @@ public TxnData coordinate(Txn txn, ConsistencyLevel consistencyLevel)
 
     private static RuntimeException throwTimeout(Txn txn, ConsistencyLevel consistencyLevel)
     {
-        throw txn.isWrite() ? new WriteTimeoutException(WriteType.TRANSACTION, consistencyLevel, 0, 0)
+        throw txn.isWrite() ? new WriteTimeoutException(WriteType.CAS, consistencyLevel, 0, 0)
```

Comment:
I am not very familiar with Accord code, but I am assuming that you use CAS here because here is no support for transaction in the client yet, or? If so, maybe its worth to add a comment.

Reply:
> no support for transaction in the client yet,

correct, java driver takes our `String` and converts to an `Enum`, which then fails; java driver does not handle unknown types...  

> If so, maybe its worth to add a comment.

makes sense, would have avoided this if a comment existed

Reply:
added

```
/**
 * Identifier for what type of operation timed out.  This type is driver facing as a String, but some drivers conver
 * this to an enum, meaning any changes to this type require protocol changes and drivers supporting
 */
```

Reply:
Since one driver will support that value and the other will not - wouldn't it be worth making it somehow dynamically adjust to the driver version?

Reply:
> wouldn't it be worth making it somehow dynamically adjust to the driver version?

I would hope we do this similar to how we downgrade to v4 or use smaller versions in internode... I would hope I provide `TRANSACTION` and there is some cql3 driver sending logic that knows that it needs to "downgrade" and hide that value...

Given that we wish to not change drivers for accord support, thought best to leave this to another ticket.